rdt-app/src/main/java/io/ona/rdt_app/fragment/RDTJsonFormFragment.java

```diff
@@ -45,16 +44,12 @@ public void onClick(View v) {
         rootView.findViewById(com.vijay.jsonwizard.R.id.next_button).setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Object isSubmit = v.getTag(R.id.submit);
-                if (isSubmit != null && Boolean.valueOf(isSubmit.toString())) {
```

Comment:
This logic still makes sense, please revert it.

It should save if submit tag exists but move to next if it doesn't.

We are modifying the logic for when the rdt test form is launched, based on answers on the registration form.

Reply:
Yeah, I merged whatever is on **master**. To move to next we'll need an additional next in the else of the `if(isSubmit != null)`